kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -2038,9 +2069,16 @@ The intrinsic gas calculation mirrors the style of the YellowPaper (appendix H).
 
 -   `#gasExec` loads all the relevant surrounding state and uses that to compute the intrinsic execution gas of each opcode.
 
+Gas cost for `TSTORE` is the same as a warm `SSTORE` of a dirty slot (i.e. original value is not new value, original is not current value, currently 100 gas).
+Gas cost of `TLOAD` is the same as a hot `SLOAD` (value has been read before, currently 100 gas).
+Gas cost cannot be on par with memory access due to transient storage’s interactions with reverts.
+
 ```k
     syntax InternalOp ::= #gasExec ( Schedule , OpCode ) [klabel(#gasExec)]
  // -----------------------------------------------------------------------
+    rule <k> #gasExec(SCHED, TLOAD _   ) => Gwarmstorageread < SCHED > ... </k>
+    rule <k> #gasExec(SCHED, TSTORE _ _) => Gwarmstorageread < SCHED > ... </k>
```

Comment:
It should be same as a warm `SSTORE`, but not storage read, right?

Reply:
Ah ok, sorry, I see that the both use 100 gas units. Maybe we could use another constant, e.g.,`Gwarmstoragedirtystore` for it to be illustrative, but not necessarily.